.github/workflows/test.yml

```diff
@@ -42,9 +43,8 @@ jobs:
         if: (! contains(matrix.python-version, 'pypy'))
         run: |
           coverage xml
-          coverage report --show-missing --fail-under=100
       - name: codecov
         if: (! contains(matrix.python-version, 'pypy'))
-        uses: codecov/codecov-action@v3
-        env:
-          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+        uses: codecov/codecov-action@v4
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
We were having coverage report upload issues, so I had to bump the version of the codecov action being used here